script/States/Application.lua

```diff
@@ -43,14 +40,12 @@ function Application:run ()
     self.resY,
     self:getWindowMode())
 
+  Config.render.gameWindow = self.window
```

Comment:
dunno if we should do that but well

Reply:
I hated doing it. The only place it's used is in HUD:onEnable(). I need to center the cursor when the HUD is enabled, but after an hour I was not able to find a handle to the game window by going through "self" in HUD.lua. That's the more correct way, and that's how I wanted to do it... but I ran out of time. If we can find the proper handle for the game window going through "self" in HUD.lua, we can eliminate creating and initializing the global.

Reply:
i´ll fix it in my game state branch